src/library/Form/AccountDropdown/Wrappers.ts

```diff
@@ -96,6 +96,7 @@ export const StyledDropdown = styled.div<any>`
       &.inactive {
         opacity: 0.5;
         cursor: default;
+        border: none;
```

Comment:
is this needed?

Reply:
yes, as outline:none is not working, this one does

Reply:
the button should just be disabled if not active, which will then make it not selectable.